src/webAppCreator.ts

```diff
@@ -426,23 +417,12 @@ class WebsiteStep extends WebAppCreatorStepBase {
                 }
             });
 
-            const fullSiteName = `${siteName}.azurewebsites.net`;
-
             // Check if the name has already been taken...
-            const nameAvailableTask = new Promise<boolean>((resolve, reject) => {
-                dns.resolve(fullSiteName, (err, addresses) => {
-                    if (err && err.code === dns.NOTFOUND) {
-                        resolve(true);
-                        return;
-                    }
-
-                    resolve(false);
-                });
-            });
+            const nameAvailability = await client.checkNameAvailability(siteName, 'site');
```

Comment:
😂👍😂

Reply:
Jay, be kind to yourself.  This is an awesome feature!